Assignments/Podcast_reflections/Question_ep1_ethos.md

```diff
@@ -0,0 +1,48 @@
+# Open Science for Physicists
+
+Especially for the course, we have recorded 4 podcast episodes extending the discussions of the course topics. 
+Participants are asked to listen to the podcast and answer some related questions. 
+
+
+## Questions for Episode 1: Ethos of Open Science and the Credibility Cycle
+
+1. What experience Erik mentions as a kickstarter for his praise for open science
+	+ Being invited to a citizen science collaboration
+	+ Extra attention for their ocean streams package after publishing it online with an open license
+	+ The opprotunity of reaching audiences outside academia 
+2. Where did Bruno Latour perform his research for the book "Laboratory Life"
+	+ Salk Institute, San Diege
+	+ Institute of Advanced Studies, Princeton
+	+ Institute Pierre and Marie Curie, Paris
+3. Which Mertonian norm for Science does Sanli specify in the podcast?
+	+ Communism
+	+ Universalism
+	+ Disinterestedness
+	+ Organized skepticism
+4. Which example is *not* mentioned by Sanli as a negative side effect of focusing on personal pursuit of credibility
+	+ Hypercompetition
+	+ Avoiding multi-disciplinary work
+	+ Avoiding team collaborations
+	+ Not publishing negative results
+5. Which of the following is not included in the credibility cycle describe by Latour and Woolgar? 
+	+ Money
+	+ Data
+	+ Societal relevance
+	+ Equipment
+	+ Publications
+6. Which school classification of open science does Sanli name first?
```

Comment:
Ay, this would be a very difficult question, if students have to answer this after they listened. Or will they see the questions before listening?